tests/helpers.js

```diff
@@ -145,3 +174,17 @@ export async function decodeSl({encodedList}) {
     'representation of a GZIP-compressed bitstring.');
   return decoded;
 }
+
+export async function addJsonAttachment(fileName, content) {
```

Comment:
I have no objections to adding json output to allure, but disabling console.log does cause concern. why is console.log disabled here?

Reply:
I'm aware it does, the reason is that if the test suite is ran with the db reporter, allure will complain that there is no allure framework loaded. It prints one line every time with this warning.

Disabling the console for this function call only was the most convenient method I found to avoid this.

This has been removed from this PR entirely we can discuss it in a subsequent PR.